src/compiler/tsbuild.ts

```diff
@@ -1788,31 +1788,25 @@ namespace ts {
 
         let reportQueue = true;
         let successfulProjects = 0;
-        let errorProjects = 0;
         while (true) {
             const invalidatedProject = getNextInvalidatedProject(state, buildOrder, reportQueue);
             if (!invalidatedProject) break;
             reportQueue = false;
             invalidatedProject.done(cancellationToken);
-            if (state.diagnostics.has(invalidatedProject.projectPath)) {
-                errorProjects++;
-            }
-            else {
-                successfulProjects++;
-            }
+            if (!state.diagnostics.has(invalidatedProject.projectPath)) successfulProjects++;
         }
 
         disableCache(state);
         reportErrorSummary(state, buildOrder);
         startWatching(state, buildOrder);
 
-        return isCircularBuildOrder(buildOrder) ?
-            ExitStatus.ProjectReferenceCycle_OutputsSkupped :
-            errorProjects ?
-                successfulProjects ?
-                    ExitStatus.DiagnosticsPresent_OutputsGenerated :
-                    ExitStatus.DiagnosticsPresent_OutputsSkipped :
-                ExitStatus.Success;
+        return isCircularBuildOrder(buildOrder)

```

Comment:
This is how we format things:
![image](https://user-images.githubusercontent.com/8052792/66845556-106e4000-ef25-11e9-905b-778b2456ccd5.png)

Reply:
Done, reluctantly.